core/src/test/java/org/jdbi/v3/core/argument/TestBeanArguments.java

```diff
@@ -58,16 +61,10 @@ public void testBindNull() throws Exception {
         verify(stmt).setNull(3, Types.NUMERIC);
     }
 
+    @RequiredArgsConstructor
+    @Getter
```

Comment:
`@Value` is equivalent

Reply:
equals/hashcode is technically a behavioral change but I guess there are extremely few tests where it will matter